ci-operator/config/openshift/ocp-release-operator-sdk/openshift-ocp-release-operator-sdk-release-4.17.yaml

```diff
@@ -22,11 +14,11 @@ images:
   from: root
   to: osdk-builder
 - dockerfile_path: release/helm/Dockerfile
-  from: base
+  from: base-rhel9
```

Comment:
Should this change be considered only when QE acknowledges all testcases are fine with rhel9 base with main branch?

Reply:
The tests already ran as part of rehearsals for both master and 4.17 branch. All the following tests were run and all of them passed:
- ci/rehearse/openshift/ocp-release-operator-sdk/master/e2e-helm
- ci/rehearse/openshift/ocp-release-operator-sdk/master/images 
- ci/rehearse/openshift/ocp-release-operator-sdk/master/sanity 
- ci/rehearse/openshift/ocp-release-operator-sdk/master/unit 
- ci/rehearse/openshift/ocp-release-operator-sdk/release-4.17/e2e-helm
- ci/rehearse/openshift/ocp-release-operator-sdk/release-4.17/images
- ci/rehearse/openshift/ocp-release-operator-sdk/release-4.17/sanity
- ci/rehearse/openshift/ocp-release-operator-sdk/release-4.17/unit